server/handler/resources.go

```diff
@@ -0,0 +1,69 @@
+package handler
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/kubernetes-sigs/kube-scheduler-simulator/resources"
+	"github.com/kubernetes-sigs/kube-scheduler-simulator/server/di"
+	"github.com/labstack/echo/v4"
+	"k8s.io/klog/v2"
+)
+
+type ResourcesHandler struct {
```

Comment:
`ResourcesHandler` is too generic a name as well. It should be renamed like service.